lintro/tools/definitions/stylelint.py

```
@@ -0,0 +1,376 @@
+        merged_options = dict(self.options)
+        merged_options.update(options)
+
+        ctx = self._prepare_execution(
+            paths,
+            merged_options,
+            no_files_message="No files to check.",
+        )
+        if ctx.should_skip:
+            return ctx.early_result  # type: ignore[return-value]
+
+        cmd = self._base_command() + ["--formatter", "json", *ctx.rel_files]
```

Ruff (0.15.20)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)